src/fast_check/range_finder.rs

```diff
@@ -1212,7 +1212,7 @@ impl<'a> PublicRangeFinder<'a> {
 fn is_module_typed(module: &crate::Module) -> bool {
   match module {
     crate::Module::Js(m) => {
-      is_typed_media_type(m.media_type) || m.maybe_types_dependency.is_some()
+      !is_untyped(&m.media_type) || m.maybe_types_dependency.is_some()
```

Comment:
Maybe we should switch to just using `is_typed_media_type` instead of `is_untyped` so that we don't have double negatives?